src/components/BookmarkCard/Thumbnail.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
+import { Link } from '../';
```

Comment:
I think this should be
```suggestion
import { Link } From '../Link';
```

to prevent possible cyclic dependencies. Everywhere it's like this of course.